portfolio/src/main/webapp/projects.html

```diff
@@ -16,16 +16,16 @@ <h1>Projects</h1>
     <p>With the help of Java and Kotlin, along with
       Android Studios I have developed the apps shown below.
     </p>
-    <div id="imageSlideshow">
-      <div class="mySlides">
+    <div id="imageslideshow">
```

Comment:
I think the convention is use a dash to separate words.
https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters